test/javascript/components/mentoring/mentor_inbox.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { reducer } from '../../../../app/javascript/components/mentoring/mentor_inbox'
```

Comment:
This feels like an implementation detail leaking out of the component. Is there perhaps a way this can be tested without exposing the reducer? CC @SleeplessByte

Reply:
Yeah, I wouldn't do this. Instead unmount and remount the component (if you are testing in isolation).

Reply:
Thanks! I'll be testing this better :)